secondary.js

```diff
@@ -32,13 +71,24 @@ const renderToDom = (divId, textToRender) => {
     firstYears.innerHTML = textToRender;
 }
 
+const filterBtn = () => {
+    const domString = `
+    <button type="button" class="btn btn-dark" id="Horned Serpent">Horned Serpent</button>
```

Comment:
This could have been created by a loop. It would make it more dynamic and easier to change. Also, you can give elements more than one id, so this has two ids. One is Horned and the other is Serpent. It would need to be connected by a character like - or _ to be defined as one id.